### PR TITLE
Dropout fix

### PR DESCRIPTION
**Pull request by George3d6**
* Added the possibility of adding a specific dropout value (from 0 to 1) in the configuration passed to `learn`

* Removed default dropout probability if there's a single input feature

### CODE REVIEW

File: lightwood/api/data_source.py [hunk 1]
```diff
@@ -17,6 +17,19 @@ def __init__(self, data_frame, configuration):
         self.encoders = {}
         self.transformer = None
         self.training = False # Flip this flag if you are using the datasource while training
+
+        self.dropout_dict = {}
+        for col in self.configuration['input_features']:
```
[the remark this answers is not in the document]
Comment (George3d6) [reply lightwood/api/data_source.py]: The `self.dropout_dict` should  store the probability to dropout each individual input column.

Are you asking what it does ? Or are you asking why not use the self.configuration object directly ? Or something else ? I'm not sure I understand your question.

**Comment by George3d6** [comment]
* Merged with master
* Added a fix for installing lightwood inside a docker container / unkonw environment
* Added dropout to configuration schema
* Added a dropout parameter into the CI testing configuration to test that It's working.

**Comment by George3d6** [comment]
Closing since CI are not running.
